.github/workflows/ci.yml

```diff
@@ -99,8 +99,36 @@ jobs:
         run: |
           python -m pip install --upgrade pip pipx
           python -m pip install -r requirements/requirements_build.txt
-      - name: Build, install and run
+      - name: Build wheel and sdist
         run: |
           python -m build && python -m twine check dist/*
-          pipx install dist/ansys_templates-0.1.dev0-py3-none-any.whl
-          pipx run --spec dist/ansys_templates-0.1.dev0-py3-none-any.whl ansys-templates --help
+
+  release:
+    name: Release
+    if: github.event_name == 'push' && contains(github.ref, 'refs/tags')
+    needs: [style, docs, tests, build]
+    runs-on: ubuntu-latest
+
+    steps:
+      - uses: actions/checkout@v2
+      - name: Set up Python
+        uses: actions/setup-python@v2
+        with:
+          python-version: 3.7
+      - name: Install dependencies
+        run: |
+          python -m pip install --upgrade pip
+          python -m pip install -r requirements/requirements_build.txt
+      - name: Publish to PyPI
+        env:
+          TWINE_USERNAME: __token__
+          TWINE_PASSWORD: $(PYPI_TOKEN)
```

Comment:
You'll have to use a github secret, as in:
https://github.com/pyansys/pymapdl/blob/0fb2494d77b3df9f3ce02f646dc0233e9cc6e503/.github/workflows/ci.yml#L300

This will use the global organization secret. Replace it with a local token at your earliest opportunity. Ping @MaxJPRey for token creation.